docs/pages/access-controls/access-monitoring.mdx

```diff
@@ -0,0 +1,427 @@
+---
+title: Getting Started with Access Monitoring
+description: Learn how to use Access Monitoring.
+---
+
+Access Monitoring is a powerful tool to understand and analyze the access patterns in your Teleport cluster.
+This guide will help you understand how to use Access Monitoring to monitor access to resources in your Teleport cluster.
+
+## Prerequisites
+- Teleport v14 or later.
+- For Self-Hosted Teleport the [AWS Athena Backend](../reference/backends.mdx#athena) is required.
+
+
+### Configuration
+
+<Tabs>
+<TabItem scope={["enterprise", "oss"]} label="Self-Hosted">
+
+To enable Access Monitoring you need to update the Teleport Auth Service configuration:
+```yaml
+auth_service:
+  access_monitoring:
+    enabled: true
+    # AWS role ARN that Teleport will assume to execute Athena SQL queries.
+    role_arn: arn:aws:iam::111111111111:role/access-monitoring-role
+    # S3 bucket where Access Monitoring reports will be stored.
+    report_results: s3://audit-long-example/report_results
```

Comment:
What is `audit-long`? Did you mean to say `audit-log`?

(Do we want people to use the same bucket they use for audit logging or a different one?)

Reply:
Yes, It was intended to be audt-log-example thanks for catching this.